manager/orchestrator/services.go

```diff
@@ -230,5 +245,12 @@ func getRunnableSlots(s *store.MemoryStore, serviceID string) (map[uint64]slot,
 		}
 	}
 
-	return runningSlots, nil
+	deadSlots := make(map[uint64]slot)
```

Comment:
If a task's DesiredState is `api.TaskStateNew`, it would be put into `deadSlots`. This isn't right.